contracts/upgradeable_contracts/OwnedUpgradeability.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity 0.4.24;
+
+import "../IOwnedUpgradeabilityProxy.sol";
+
+
+contract OwnedUpgradeability {
+
+    function upgradeabilityAdmin() public view returns (address) {
+        return IOwnedUpgradeabilityProxy(this).proxyOwner();
+    }
+
+    modifier onlyProxyOwner() {
```

Comment:
I would avoid of using the same name for different implementation of modifiers since it could to issues in the future. May we have the name like 'onlyIfOwnerOfProxy' and add a comment in here why we need two different modifier for the same functionality?

Reply:
Thanks, Fixed

Reply:
I am OK with the changes.